operations/helm/charts/mimir-distributed/templates/distributor/distributor-svc-headless.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
     {{- end }}
   annotations:
-    {{- toYaml .Values.distributor.service.annotations | nindent 4 }}
+    {{- toYaml .Values.distributor.service.headlessAnnotations | nindent 4 }}
```

Comment:
This is a breaking change. The setups that are currently in production, will stop working after they update to this one.